hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/state/Utils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.web3.state;
+
+import static com.hedera.mirror.web3.evm.utils.EvmTokenUtils.entityIdNumFromEvmAddress;
+
+import com.hedera.hapi.node.base.ContractID;
+import com.hedera.mirror.common.domain.entity.EntityId;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+import jakarta.annotation.Nullable;
+import lombok.CustomLog;
+import lombok.experimental.UtilityClass;
+import org.hyperledger.besu.datatypes.Address;
+
+@CustomLog
+@UtilityClass
+public class Utils {
+
+    public static Address convertPbjBytesToBesuAddress(final Bytes bytes) {
+        final var evmAddressBytes = bytes.toByteArray();
+        return Address.wrap(org.apache.tuweni.bytes.Bytes.wrap(evmAddressBytes));
+    }
+
+    @Nullable
+    public static EntityId convertContractIDToEntityId(final ContractID contractID) {
```

Comment:
Use a simpler name so we can have many overloaded variants for each pbj type.

```suggestion
    public static EntityId toEntityId(final ContractID contractID) {
```

Reply:
Renamed